examples/symfony.php

```diff
@@ -45,6 +46,13 @@ public function json()
 
         return print_r($payload, true);
     }
+
+    public function valuesArray()
+    {
+        $name = $this->queryParameter('arr')->commaSeparated()->int();
```

Comment:
Maybe the example could be more real life, e.g. calling the `arr` parameter `userIds` instead